crates/nu-command/src/conversions/into/bool.rs

```diff
@@ -0,0 +1,183 @@
+use nu_engine::CallExt;
+use nu_protocol::{
+    ast::{Call, CellPath},
+    engine::{Command, EngineState, Stack},
+    Category, Example, PipelineData, ShellError, Signature, Span, SyntaxShape, Value,
+};
+
+#[derive(Clone)]
+pub struct SubCommand;
+
+impl Command for SubCommand {
+    fn name(&self) -> &str {
+        "into bool"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("into bool")
+            .rest(
+                "rest",
+                SyntaxShape::CellPath,
+                "column paths to convert to binary (for table input)",
```

Comment:
I think the comment should say `to boolean` instead of `to binary`

Reply:
Oh! thanks for finding my mistake 😅